config/config.go

```diff
@@ -64,7 +65,7 @@ var DefaultDiscConfig DiscConfig = DiscConfig{
 	TCP:        9000,
 	DBPath:     "",
 	ForkDigest: [4]byte{0x6a, 0x95, 0xa1, 0xa9},
-	LogPath:    "nodes.log",
+	LogPath:    "peer_discovered.log",
```

Comment:
Changed path to be consistent with event name

Reply:
Let's rename it to the table names, i.e. `discovery_events.log`

Reply:
done!